src/brutus-module-weather/brutus_module_weather/tests/cloudQuestion_test.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+from flask import json
+
+from .common import BrutusTestCase
+
+
+class CloudTestCase(BrutusTestCase):
+    """
+    Simple tests for the weather module
+    Check that questions about
+        clouds return the correct answers
+    """
+
+    cloudQuestions = ['Is it cloudy',
+                      'Are there a lot of clouds'
+                      'How many clouds are there'
```

Comment:
Good catch. I had already merged and deleted the branch so it is fixed in this pull request #39